testutil/simibc/relay_util.go

```diff
@@ -7,8 +7,8 @@ import (
 	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
 	host "github.com/cosmos/ibc-go/v3/modules/core/24-host"
 	ibctmtypes "github.com/cosmos/ibc-go/v3/modules/light-clients/07-tendermint/types"
-	ibctesting "github.com/cosmos/ibc-go/v3/testing"
-	"github.com/cosmos/ibc-go/v3/testing/simapp"
+	simapp "github.com/cosmos/interchain-security/ibc/simapp"
```

Comment:
@danwt, not important for this PR but we should try not to rely on any simapp related packages

Reply:
I think technically this doesn't even have to live in simapp...and with v0.47 of the sdk this will be possible for any helper function

Reply:
Hey not sure I follow, sent a DM

Reply:
Just a note: this is the ibc-go simapp, not the SDK one. But maybe they have the same reason to avoid.